src/components/ProtectedRoute.jsx

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import {Navigate} from 'react-router-dom';
+
+const ProtectedRoute = ({user, Children}) => {
```

Comment:
In `App.js`, you are passing in `component` and `isAuth` as props, what gives?

Reply:
I don't really understand your commemnt

Reply:
Your latest commit fixed this, passing in `user` instead of `isAuth`. Though I think the latter is a better name.

On another note, `Children` should be lowercase here